### PR TITLE
Repo config should be url decoded

### LINKED ISSUE #39: NPE on repo update
When trying to update or create repo via dashboard, management API throws NPE and returns 500 status:
```
java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "com.amihaiemil.eoyaml.YamlMapping.value(String)" because "repo" is null
java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "com.amihaiemil.eoyaml.YamlMapping.value(String)" because "repo" is null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException: Cannot invoke "com.amihaiemil.eoyaml.YamlMapping.value(String)" because "repo" is null
	at com.artipie.management.api.ApiRepoUpdateSlice.lambda$response$2(ApiRepoUpdateSlice.java:118)
	... 4 more
```

Artipie version is `v0.20`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #39 
Body with repo configs should be url-decoded.

### CODE REVIEW

**Review by genryxy — DISMISSED** [review]
@olenagerasimova thanks, looks good

**Comment by olenagerasimova** [comment]
@g4s8 please, merge

File: src/test/java/com/artipie/management/api/ApiRepoUpdateSliceTest.java [hunk 1]
```diff
@@ -131,7 +133,10 @@ void updatesRepoConfiguration(final String extension, final boolean defstor)
     }
 
     private static byte[] body(final String reponame, final String type, final boolean defstor) {
-        return String.format("repo=%s&config=%s", reponame, yaml(type, defstor)).getBytes();
+        return URLDecoder.decode(
```
Comment (g4s8) [inline src/test/java/com/artipie/management/api/ApiRepoUpdateSliceTest.java:136]: @olenagerasimova I suppose the request body should be encoded, not decoded?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check one comment

**Comment by olenagerasimova** [comment]
@g4s8 my bad, thanks, corrected

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**